sc2mqtt.py

```diff
@@ -371,27 +377,44 @@ async def updateValues(self, mqttc):
                         if "calc" in self.statusValues[stateId]:
                             state["value"] = self.statusValues[stateId]["calc"](state["value"])
                         _LOGGER.info("%s -> %s(%s)" %(self.statusValues[stateId]["statusName"], state["textId"], state["value"]))
-                        stopic = "skoda2mqtt/%s_%s/STATE"% (vin, self.statusValues[stateId]["statusName"])
+                        stopic = "skoda2mqtt/%s/%s/STATE"% (vin, self.statusValues[stateId]["statusName"])
```

Comment:
You cannot nest that much if you want this to be working with Home Assistant. I was not able to get anything with more than one nesting level (skoda2mqtt/<measurement_name>/STATE) working. Please revert this before I can merge the PR.

Reply:
I have it running that way some months (HA 0.115 till HA 0.118)
What I have found is if the discovery topic is not retained then the HA has to be started before this starts.

I would change the retain state of the discovery topic.